cypress/integration/loginAngular.spec.js

```diff
@@ -0,0 +1,34 @@
+import { VAR, URL } from '../const/constants';
+import { IDs } from '../page-objects/loginAngular';
+import{ HAVETEXT } from '../const/helpers';
```

Comment:
```suggestion
import { HAVETEXT } from '../const/helpers';
```
add a space after `import`

Reply:
Change in the branch "NewArticule"